src/squirrel/supertux_api.cpp

```diff
@@ -872,6 +904,9 @@ void register_supertux_scripting_api(ssq::VM& vm)
   level.addFunc("toggle_pause", &scripting::Level::toggle_pause);
   level.addFunc("pause_target_timer", &scripting::Level::pause_target_timer);
   level.addFunc("resume_target_timer", &scripting::Level::resume_target_timer);
+  level.addFunc("override_item_pocket", &scripting::Level::override_allow_item_pocket);
```

Comment:
The C++ function name is different. Make sure they both stay the same for accuracy in documentation.

```suggestion
  level.addFunc("override_item_pocket", &scripting::Level::override_item_pocket);
```